src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -498,14 +521,14 @@ class BerdyHelper
     /**
      * Resize and set to zero Berdy matrices.
      */
-    bool resizeAndZeroBerdyMatrices(MatrixDynSize & D, VectorDynSize & bD,
-                                    MatrixDynSize & Y, VectorDynSize & bY);
+    bool resizeAndZeroBerdyMatrices(SparseMatrix &D, VectorDynSize &bD,
+                                    SparseMatrix &Y, VectorDynSize &bY);
 
     /**
      * Get Berdy matrices
      */
-    bool getBerdyMatrices(MatrixDynSize & D, VectorDynSize & bD,
-                          MatrixDynSize & Y, VectorDynSize & bY);
```

Comment:
Is it ok to create a temporary sparse matrix to hold the result before the conversion?